stdlib/scanf.mli

```diff
@@ -247,6 +250,11 @@ val bscanf : Scanning.in_channel -> ('a, 'b, 'c, 'd) scanner
     string.
 *)
 
+val bscanf_opt : Scanning.in_channel -> ('a, 'b, 'c, 'd) scanner_opt
+(** Same as {!Scanf.bscanf}, but returns [None] in case of scanning failure.
+
+    @since 5.00.0 *)
```

Comment:
Answering this question is above my paygrade, but: is this the right version number?

Reply:
The right version number should be `5.0`.

Reply:
Thanks, fixed.